### PR TITLE
chore: bump deps

### PR DESCRIPTION
**Pull request by DmitriyMV**
Bump stuff:
- bump bldr
- bump binutils to 2_41
- bump mpfr to 4.2.1
- bump linux to 6.1.60
- bump gcc to 13.2.0
- bump actions/checkout to v4
- bump docker/login-action to v3

### CODE REVIEW

File: Pkgfile [hunk 1]
```diff
@@ -6,19 +6,19 @@ vars:
...
   # renovate: datasource=git-tags extractVersion=^binutils-(?<version>.*)$ depName=git://sourceware.org/git/binutils-gdb.git
-  binutils_version: 2_40
-  binutils_sha256: 0f8a4c272d7f17f369ded10a4aca28b8e304828e95526da482b0ccc4dfc9d8e1
-  binutils_sha512: a37e042523bc46494d99d5637c3f3d8f9956d9477b748b3b1f6d7dfbb8d968ed52c932e88a4e946c6f77b8f48f1e1b360ca54c3d298f17193f3b4963472f6925
+  binutils_version: 2_41
+  binutils_sha256: ae9a5789e23459e59606e6714723f2d3ffc31c03174191ef0d015bdf06007450
+  binutils_sha512: 5df45d0bd6ddabdce4f35878c041e46a92deef01e7dea5facc97fd65cc06b59abc6fba0eb454b68e571c7e14038dc823fe7f2263843e6e627b7444eaf0fe9374
 
   # renovate: datasource=git-tags extractVersion=^releases/gcc-(?<version>.*)$ depName=git://gcc.gnu.org/git/gcc.git
-  gcc_version: 12.3.0
```
Comment (DmitriyMV) [inline Pkgfile:14]: @frezbo is it safe to do?

File: Pkgfile [hunk 2]
```diff
@@ -6,19 +6,19 @@ vars:
...
-  gcc_sha256: 949a5d4f99e786421a93b532b22ffab5578de7321369975b91aec97adfda8c3b
-  gcc_sha512: 8fb799dfa2e5de5284edf8f821e3d40c2781e4c570f5adfdb1ca0671fcae3fb7f794ea783e80f01ec7bfbf912ca508e478bd749b2755c2c14e4055648146c204
+  gcc_version: 13.2.0
```
Comment (DmitriyMV) [inline Pkgfile:14]: @frezbo is it safe to do?

Comment (frezbo) [reply Pkgfile:14]: well have to build tools and pkgs to see if all builds successfully

Comment (DmitriyMV) [reply Pkgfile]: Tested locally with both `make tools PLATFORM=linux/amd64` and `make all PLATFORM=linux/amd64` for pkgs repository. Both compiled, so we should be good?

**Review by frezbo — APPROVED** [review]

**Pull request merged**